src/ngraph/frontend/onnx_import/default_opset.hpp

```diff
@@ -0,0 +1,6 @@
+#include "ngraph/opsets/opset1.hpp"
+
+namespace ngraph
```

Comment:
Ok, I can change it. But do we still want to use in code full namespace `ngraph::onnx_import::default_opset::Add` or rather `default_opset::Add` because we actually are in  `ngraph::onnx_import` namespace ?

Reply:
I think we should use the short form `default_opset::Add` wherever possible.

Reply:
Done